cip/1.accepted/CIP2015-05-13-EXISTS.adoc

```diff
@@ -156,7 +234,7 @@ RETURN person
 
 This proposal also allows for powerful subqueries, for example using aggregation inside the `EXISTS {}` query.
 
-.Find all teams that have at least two members who have worked on successful projects.
+Find all teams that have at least two members who have worked on successful projects.
```

Comment:
how come you removed the leading dot?

Reply:
Do not remember exactly, but think it was not used consistently in similar spots.

Reply:
OK. It is used to render above the listing (the Cypher code in this case) in a special way (not just a paragraph). I kinda like it but no objection here.